sre-cloud.md

```diff
@@ -0,0 +1,71 @@
+# S.R.E / Cloud Challenge
+
+## About the Challenge... 
+
+We have two challenges, the first one is an intermediate test, and the second is a more advanced: The both challenges is about defining an architecture for Zé at AWS, considering several requirements that are necessary to consider.
+
+Feel free to ask any question - we really want you to succeed and then, join us to help evolve our platform.
+
+## The First Challenge - Intermediate
+
+This first one is build an simple infrastructure as code (AWS). 
+
+### Specifications: 
+
+- An nginx presenting a static web page
+- Allow this application to be publicly accessible
+- Apply as best practices thinking about security, high availability, failure and costs
+- Develop automation with your preferred tool to allow:
+- Deploy the new version of the web application without downtime
+- Rollback to previous version when needed
+- Take good care of the Readme, explaining how your solution was designed and how to deploy it
+
+
+### Advice
+
+- **Try to design and implement your solution as you would do for real production code**. Even though the test is simple, do it as robustly and professionally as you can.
+Points may have a much-needed ability to make technical decisions.
+
+If in doubt, do not hesitate to ask us.
+
+### How to deliver it
+
+Please, use Github to host your code and add @ze-engineering-code-challenge as a collaborator for it. This Github account (@ze-engineering-code-challenge) is solely used by Zé's engineers to download your code and review it.
+
+**Once you have finished the challenge, please submit your information on this amazing form:**
+  https://docs.google.com/forms/d/e/1FAIpQLSePVCWxsHZHoRqJY9-XFJHuL7iOjO00sfhZksLBmDbR0KuoLg/viewform
+
+## The Second Challenge - Advanced
+
+This second challenge is more like a case than a code challenge itself.
+
+### The Story
+
+Imagine that we are starting Zé Delivery at this moment and we are recruiting the team and finally starting the development of Zé's products, which are: 
```

Comment:
Acho desnecessário falar que estamos recrutando a equipe. Eu cortaria. Deixaria algo como "Imagine we are starting to build the very first version of Zé Delivery platform, considering:"